### PR TITLE
Upgrade Serilog to version 2.0

### PR DESCRIPTION
**Pull request by mkrain**
I've upgraded the Serilog package and added the new File and Console sink nuget packages.  Running locally I see that 4 tests are failing, however, this was the case when cloning the original repository.

### CODE REVIEW

File: NServiceBus.Serilog.Tracing/app.config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SimonCropp) [inline NServiceBus.Serilog.Tracing/app.config]: should not need this

File: Sample/app.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SimonCropp) [inline Sample/app.config]: should not need this

File: Tests/app.config [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SimonCropp) [inline Tests/app.config]: should not need this

File: TracingSample/TracingSample.csproj [hunk 4]
```diff
@@ -38,17 +38,21 @@
...
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Serilog.Sinks.File, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10, processorArchitecture=MSIL">
+      <HintPath>..\packages\Serilog.Sinks.File.2.2.0\lib\net45\Serilog.Sinks.File.dll</HintPath>
+      <Private>True</Private>
     </Reference>
     <Reference Include="Serilog.Sinks.Seq, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Serilog.Sinks.Seq.1.5.5\lib\net45\Serilog.Sinks.Seq.dll</HintPath>
+      <HintPath>..\packages\Serilog.Sinks.Seq.1.5.36\lib\net45\Serilog.Sinks.Seq.dll</HintPath>
```
Comment (SimonCropp) [inline TracingSample/TracingSample.csproj]: shouldnt we also update the seq nuget to 2.0

File: TracingSample/app.config [hunk 5]
```diff
@@ -4,11 +4,11 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Serilog" publicKeyToken="24c2f752a8e58a10" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-1.3.0.0" newVersion="1.3.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-2.0.0.0" newVersion="2.0.0.0" />
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Serilog.FullNetFx" publicKeyToken="24c2f752a8e58a10" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-1.3.0.0" newVersion="1.3.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-1.4.0.0" newVersion="1.4.0.0" />
```
Comment (SimonCropp) [inline TracingSample/app.config]: this looks wrong

**Comment by SimonCropp** [comment]
@mkrain yeah the failing tests are actually a bug in the NSB core acceptance tests. they have been removed in a newer version of NSB nugets. they will be fixed in the move to NSB 6

a few minor comments re binding redirects and Seq. if u want to fix those i will deploy

File: NServiceBus.Serilog.Tracing/app.config (hunk 1 above)
Comment (mkrain) [inline NServiceBus.Serilog.Tracing/app.config]: I'll remove both app.configs, they were' added automatically by nuget when the packages were updated.  Probably early am tomorrow when I get in the office.

File: TracingSample/TracingSample.csproj (hunk 4 above)
Comment (mkrain) [inline TracingSample/TracingSample.csproj]: I'll see if there's a newer package and update it as well.

File: TracingSample/app.config (hunk 5 above)
Comment (mkrain) [inline TracingSample/app.config]: This was done automatically, should I revert it?  Normally I don't like having bindingRedirect but I assumed VS new what it was doing.

**Comment by mkrain** [comment]
I made the requested changes.  Do you want me to rebase?

**Comment by SimonCropp** [comment]
No need to rebase. I will try to release this tonight

**Comment by mkrain** [comment]
Thanks.  If you can get a nuget package out there I would really appreciate it.

**Pull request merged**